config/initializers/ezid_client.rb

```diff
@@ -0,0 +1,6 @@
+Ezid::Client.configure do |config|
+  config.default_shoulder = "ark:/99999/fk4"
+  config.user = "apitest"
+  config.password = "apitest"
```

Comment:
I think you are right. Didn't realize that the test user password is not public available information. I will remove it, with a new forced push to remove the history, and add README